pages/dkp/kommander/2.2/clusters/attach-cluster/requirements-for-attaching/index.md

```diff
@@ -72,6 +72,10 @@ To attach an existing EKS cluster, refer to the specific information in [Attach
 Consider the additional resource requirements for running the platform services you want Kommander to manage, and ensure that your existing clusters comply.
 -->
 
+## Attach clusters with an existing cert-manager installation
+
+If you are attaching clusters that already have cert-manager installed, the cert-manager HelmRelease provided by Kommander will fail to deploy, due to your existing cert-manager installation. As long as the pre-existing cert-manager functions as expected, you can ignore this.
+
```

Comment:
Per today's standup, it's probably worth calling out explicitly that a Konvoy cluster will have cert-manager installed, and therefore this will occur. Maybe something like this?

```
<p class="message--note"><strong>NOTE: </strong>Clusters provisioned using Konvoy have cert-manager installed. Kommander's cert-manager `HelmRelease` will fail to deploy on a Konvoy cluster.</p>
```

Reply:
how does this look?

Reply:
Pushing a change for styles, that's all